1-js/02-first-steps/17-javascript-specials/article.md

```diff
@@ -1,234 +1,238 @@
 
-Variables are dynamically typed. They can store any value:
+Zmienne są typowane dynamicznie. Mogą przechowywać dowolną wartość:
 
 ```js
 let x = 5;
-x = "John";
+x = "Jan";
 ```
 
-There are 7 data types:
+Istnieje 8 typów danych:
 
-- `number` for both floating-point and integer numbers,
-- `string` for strings,
-- `boolean` for logical values: `true/false`,
-- `null` -- a type with a single value `null`, meaning "empty" or "does not exist",
-- `undefined` -- a type with a single value `undefined`, meaning "not assigned",
-- `object` and `symbol` -- for complex data structures and unique identifiers, we haven't learnt them yet.
+- `number` zarówno dla liczb zmiennoprzecinkowych, jak i całkowitych,
+- `bigint` dla liczb całkowitych dowolnej wielkości,
+- `string` dla ciągów znaków,
+- `boolean` dla wartości logicznych (prawda/fałsz): `true/false`,
+- `null` - typ przyjmujący tylko wartość `null`, znaczący "pusty" albo "nie istnieje",
+- `undefined` - typ przyjmujący tylko wartość `undefined`, znaczący "nieprzypisany",
+- `object` i `symbol` - wykorzystywany w złożonych strukturach danych i unikalnych identyfikatorach, ale jeszcze się ich nie nauczyliśmy.
 
-The `typeof` operator returns the type for a value, with two exceptions:
+Operator `typeof` zwraca typ wartości, z dwoma wyjątkami:
 ```js
-typeof null == "object" // error in the language
-typeof function(){} == "function" // functions are treated specially
+typeof null == "object" // wewnętrzny błąd w języku
+typeof function(){} == "function" // funkcje są traktowane specjalnie
 ```
 
-More in: <info:variables> and <info:types>.
+Więcej w: <info:variables> i <info:types>.
 
-## Interaction
+## Interakcje
 
-We're using a browser as a working environment, so basic UI functions will be:
+Używamy przeglądarki jako środowiska pracy, więc podstawowe funkcje interfejsu użytkownika będą następujące:
 
-[`prompt(question, [default])`](mdn:api/Window/prompt)
-: Ask a `question`, and return either what the visitor entered or `null` if they clicked "cancel".
+[`prompt(pytanie, [wartość domyślna])`](mdn:api/Window/prompt)
+: Zadaje `pytanie`, i zwraca albo to, co wpisał użytkownik lub `null` jeżeli kliknął "anuluj".
 
-[`confirm(question)`](mdn:api/Window/confirm)
-: Ask a `question` and suggest to choose between Ok and Cancel. The choice is returned as `true/false`.
+[`confirm(pytanie)`](mdn:api/Window/confirm)
+: Zadaje `pytanie` i umożliwia wybór pomiędzy Ok i Anuluj. Wybór jest zwracany w postaci `true/false`.
 
-[`alert(message)`](mdn:api/Window/alert)
-: Output a `message`.
+[`alert(wiadomość)`](mdn:api/Window/alert)
+: Wyświetla `wiadomość`.
 
-All these functions are *modal*, they pause the code execution and prevent the visitor from interacting with the page until they answer.
+Wszystkie te funkcje są *modalne* (w postaci okien dialogowych), wstrzymują wykonywanie kodu i uniemożliwiają odwiedzającemu interakcję ze stroną, dopóki nie odpowie.
```

Comment:
Please don't add any special phrases like this one: "(w postaci okien dialogowych)"